### ISSUE TITLE
readmacro could not read #f(nan inf -inf)

### ISSUE THREAD

**Issue by k-okada** [euslisp/EusLisp#161]
https://github.com/euslisp/EusLisp/issues/149#issuecomment-162134160

**Comment by k-okada** [comment]
`*nan*, *inf*` is defined in https://github.com/euslisp/EusLisp/blob/master/lisp/geo/primt.l#L161,

`(setq v (float-vector (/ 1.0 0.0) (/ -1.0 0.0) (/ 0.0 0.0)))` outputs `#f(inf -inf -nan)` because libc? output if the float value is nan or inf.

So if you run codes like
```
(setq a (/ 1.0 0.0))
(dump-loadable-structure "/tmp/test-a-dump-structure.l" a)
```
it outputs like
```
(setq a 'inf)
```
this mean if we define `inf` instaad of `*inf*` , we can read this value

In order to read `#f(inf)` we have to change something linke #162

**Comment by k-okada** [comment]
closed via #162
